x-pack/plugins/search_inference_endpoints/public/components/endpoints_router.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { Route, Routes } from '@kbn/shared-ux-router';
+
+import { INFERENCE_ENDPOINTS_PATH } from './routes';
+
+import { InferenceEndpoints } from './inference_endpoints';
+
+export const EndpointsRouter: React.FC = () => {
+  return (
+    <Routes>
+      <Route exact path={INFERENCE_ENDPOINTS_PATH}>
+        <InferenceEndpoints />
```

Comment:
The layout of this in serverless is wrong. We need to wrap this in a PageTemplate or something: 
![image](https://github.com/elastic/kibana/assets/1972968/85f34837-0e65-4409-b6d5-6e9baf4ed3c0)